app/base/static/assets/js/vis/simple-line-chart.js

```diff
@@ -133,5 +141,46 @@ class SimpleLineChart {
             focus.select(".x-hover-line").attr("y2", height - y(d[field]));
             focus.select(".y-hover-line").attr("x2", width + width);
         }
+
+        //declare resize function
+        function resize() {
+
+            let w=window.innerWidth - 260 - margin.left - margin.right;
```

Comment:
Same here - we have the number 260. It may be better to define a set of constants in common.js.

Reply:
same with bar chart, replace it with CHART_WIDTH = document.getElementById('charts').offsetWidth